ansible/roles/base/tasks/main.yml

```diff
@@ -81,6 +80,32 @@
       - python-apt
     state: present
 
+- when: base__managed_ntpd | default(True)
+  block:
+    - name: check that ISC ntpd is not installed
+      apt:
+        name: ntp
+        state: absent
+        purge: yes
+
+    - name: install openntpd on bare metal
```

Comment:
the name suggests it will only be installed on bare metal but this is no longer true right?

Reply:
Indeed!